### PR TITLE
feat: add a maven archetype module

### PR DESCRIPTION
**Pull request by hantsy**

### CODE REVIEW

**Comment by hantsy** [comment]
There are some questions about the maven archetype module.
1. I need to setup the parent/child relation in this project?
2. A better name for this module? 
3. The groupId is `org.eclipse`, it is better to use `jakarta.starter` or  `org.eclipse.ee4j.starter`, I am not sure.
4. CI for build the project. [My original project](https://github.com/hantsy/maven-archetype-jakartaee9) is passed the build tests in Github Actions.

File: maven-archetype-jakartaee9/src/main/resources/META-INF/maven/archetype-metadata.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
```
Comment (ivargrimstad) [inline maven-archetype-jakartaee9/src/main/resources/META-INF/maven/archetype-metadata.xml:4]: The Starter for Jakarta EE is under EPL-2.0. Any license headers must match this license.

**Review by ivargrimstad — CHANGES_REQUESTED** [review]
The license header(s) must be fixed

**Comment by ivargrimstad** [comment]
> 3. The groupId is `org.eclipse`, it is better to use `jakarta.starter` or  `org.eclipse.ee4j.starter`, I am not sure.
The `jakarta` namespace is reserved for Jakarta EE specifications, so `org.eclipse.starter` is probably the most correct to use.

File: maven-archetype-jakartaee9/src/main/resources/META-INF/maven/archetype-metadata.xml (hunk 1 above)
Comment (hantsy) [reply maven-archetype-jakartaee9/src/main/resources/META-INF/maven/archetype-metadata.xml]: Clean license headers of all files.

**Comment by agoncal** [comment]
Hi. Any progress on that ? It would be great to have a Maven archetype to bootstrap a Jakarta EE 9 project.

**Comment by m-reza-rahman** [comment]
Yes, PR forthcoming.

**Comment by m-reza-rahman** [comment]
I think this PR can be withdrawn now @hantsy.

**Comment by m-reza-rahman** [comment]
@ivargrimstad, I would suggest closing this PR. We've added Maven Archetypes now.

**Pull request closed without merging**